### ISSUE TITLE
lsdef of nicips.<interface> attributes include regular expressions

### ISSUE THREAD

**Issue by mfoliveira** [xcat2/xcat-core#476]
Notice 2 things:
1) the nicips attribute for the *first* interface incorporated the regex first field "|p8r(\d+)n(\d+)|" in the  interface name.
2) the nicips attribute for the *last* interface shows the regex trailing "|" in the value field.

```
	[root@xcat-mn ~]# lsdef -t group s822lc
	Object name: s822lc
	<...>
	    nicips.|p8r(\d+)n(\d+)|enP3p3s0f0=10.3.($1+0).($2+0)
	    nicips.enP3p3s0f1=10.4.($1+0).($2+0)
	    nicips.ib0=10.5.($1+0).($2+0)
	    nicips.ib1=10.6.($1+0).($2+0)|
	<...>
```

This is the command used for definition:

```
	[root@xcat-mn ~]# chdef -t group s822lc \                                                                                                                                                              
	>    nictypes='enP3p3s0f0!Ethernet,enP3p3s0f1!Ethernet,ib0!Infiniband,ib1!Infiniband' \
	>    nicnetworks='enP3p3s0f0!net-app-10gbe-port1,enP3p3s0f1!net-app-10gbe-port2,ib0!net-app-ib-port1,ib1!net-app-ib-port2' \
	>    nichostnamesuffixes='enP3p3s0f0!-10gbe-1,enP3p3s0f1!-10gbe-2,ib0!-ib-1,ib1!-ib-2' \
	>    nicips='|p8r(\d+)n(\d+)|enP3p3s0f0!10.3.($1+0).($2+0),enP3p3s0f1!10.4.($1+0).($2+0),ib0!10.5.($1+0).($2+0),ib1!10.6.($1+0).($2+0)|'
	1 object definitions have been created or modified.
```

The values for a member of that group, all correct:

```
	[root@xcat-mn ~]# lsdef p8r1n1
	Object name: p8r1n1
	<...>
	    groups=all,s822lc
	<...>
	    nicips.enP3p3s0f0=10.3.1.1
	    nicips.enP3p3s0f1=10.4.1.1
	    nicips.ib0=10.5.1.1
	    nicips.ib1=10.6.1.1
	<...>
```

**Comment by neo954** [comment]
Really cool.

**Comment by neo954** [comment]
I think may be the ``+0`` can be omit. Even if there are prefix ``0``s in the IPv4 address, it is okay.

**Comment by daniceexi** [comment]
This issue only impacts the output of group definition, no impact to the real function of node definition. Will try to fix in 2.12.1.

**Comment by daniceexi** [comment]
@mfoliveira  The issue has been fixed in #1308. Please take a look.

**Comment by daniceexi** [comment]
Did not get reply from @mfoliveira, close it here. If it recreated it, please reopen.
